### DISCUSSION TITLE
microphone not authorized

### DISCUSSION THREAD

**Discussion by chenqp** [langgenius/dify#5248] [Help]
I have enable the Speech to Text function, can see the voice input icon in the input box on the web application side. Click the icon to convert the voice input into text,but : microphone not authorized .
![1](https://github.com/langgenius/dify/assets/8595915/d10b0ef5-0257-474b-8b74-2bbc1f760fac)
Chrome has been granted permissions
![2](https://github.com/langgenius/dify/assets/8595915/b7897520-a711-4dc7-a73f-51fc7c056cb3)

**Comment by Yash-1511** [comment]
It will not work in http.
